packages/block-template/package.json

```
@@ -26,7 +26,8 @@
     "dev": "concurrently -n webpack,webpack-dev-server -c green,cyan \"yarn build:dev --watch\" \"yarn run-dev-server\"",
     "lint:tsc": "tsc --noEmit",
     "run-dev-server": "cross-env NODE_ENV=development webpack-dev-server --config webpack-dev-server.config.js",
-    "schema": "typescript-json-schema tsconfig.json AppProps --required true --out dist/block-schema.json"
+    "schema": "typescript-json-schema tsconfig.json AppProps --required true --out dist/block-schema.json",
+    "serve": "npx serve -C -l 61234 ./dist/"
```

```suggestion
    "serve": "npx serve --cors --listen 61234 ./dist/"
```

[optional]

Commands in package.json scripts, markdown blocks as well as `execa` args are more often read than written. Avoiding shortcuts makes them easier to understand as there’s no need to refer to the manual.